econet_hvac_furnace.yaml

```diff
@@ -108,9 +108,10 @@ sensor:
     device_class: "current"
     entity_category: "diagnostic"
     accuracy_decimals: 1
-    unit_of_measurement: "uA"
+    unit_of_measurement: "mA"
     filters:
       - delta: 0.09
+      - multiply: .001 # Convert from uA to mA
```

Comment:
Since this is a template sensor this could go where this is calculated. Does the delta need to be adjusted?

Reply:
Filters are run in order, so the Delta wouldn't need to be adjusted if implemented as shown. If I move it the Delta would need to be adjusted.

This is where it's set: https://github.com/esphome-econet/esphome-econet/blob/318f261d7a87711a2e46be3cbcffb08f90487c5b/econet_hvac_furnace.yaml#L32

@stockmopar what's the reason we divide by 10 there? Does it provide the current in 1/10 uA units by default?

Reply:
Yep it does provide current to tenths of uA.